src/examples/mfem_common_plane.cpp

```diff
@@ -0,0 +1,429 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+#include <set>
+
+#ifdef TRIBOL_USE_UMPIRE
+// Umpire includes
+#include "umpire/ResourceManager.hpp"
+#endif
+
+// MFEM includes
+#include "mfem.hpp"
+
+// Axom includes
+#include "axom/CLI11.hpp"
+#include "axom/slic.hpp"
+
+// Redecomp includes
+#include "redecomp/redecomp.hpp"
+
+// Tribol includes
+#include "tribol/common/Parameters.hpp"
+#include "tribol/config.hpp"
+#include "tribol/interface/tribol.hpp"
+
+/// Simple central difference method.
+class CentralDiffSolver : public mfem::SecondOrderODESolver
+{
+public:
+  CentralDiffSolver(const mfem::Array<int>& bc_vdofs_)
+  : bc_vdofs { bc_vdofs_ },
+    first_step { true } {}
+
+  void Step(mfem::Vector& x, mfem::Vector& dxdt, double& t, double& dt) override
+  {
+    // acceleration at t
+    f->SetTime(t);
+    if (first_step)
+    {
+      accel.SetSize(x.Size());
+      f->Mult(x, dxdt, accel);
```

Comment:
See comment on line 362. I see that this class (or base class rather) is initialized with your explicit mechanics solver. Is this call to f->Mult() actually invoking the elasticity integrator etc in the explicit mechanics class under the hood? A simple comment for clarity on how/where these two classes interact would be nice.

Reply:
It calls the update stored in f, the `mfem::SecondOrderTimeDependentOperator` linked through the `Init()` method.  Comment has been added.